prql-compiler/prqlc/README.md

```diff
@@ -32,3 +40,46 @@ FROM
 WHERE
   has_dog
 ```
+
+PRQL query can be executed with CLI tools working with SQL, such as
+[DuckDB CLI](https://duckdb.org/docs/api/cli.html).
+
+```sh
+$ curl -sL https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv -o penguins.csv
+$ echo "from `penguins.csv` | take 3" | prqlc compile | duckdb
+┌─────────┬───────────┬────────────────┬───────────────┬───────────────────┬─────────────┬─────────┐
+│ species │  island   │ bill_length_mm │ bill_depth_mm │ flipper_length_mm │ body_mass_g │   sex   │
+│ varchar │  varchar  │     double     │    double     │       int64       │    int64    │ varchar │
+├─────────┼───────────┼────────────────┼───────────────┼───────────────────┼─────────────┼─────────┤
+│ Adelie  │ Torgersen │           39.1 │          18.7 │               181 │        3750 │ MALE    │
+│ Adelie  │ Torgersen │           39.5 │          17.4 │               186 │        3800 │ FEMALE  │
+│ Adelie  │ Torgersen │           40.3 │          18.0 │               195 │        3250 │ FEMALE  │
+└─────────┴───────────┴────────────────┴───────────────┴───────────────────┴─────────────┴─────────┘
+```
+
+Executing this command without any argument will start interactive mode,
+allowing you to write PRQL query interactively. In this mode, after you write
+PRQL and press `Ctrl-D` (Linux, macOS) or `Ctrl-Z` (Windows) to display the
+compiled SQL.
+
+```sh
+$ prqlc compile
+```
+
+As with using it as a filter, you can pass the SQL string output to the DuckDB
+CLI, etc.
+
+```sh
+$ prqlc compile | duckdb
+from `penguins.csv`
+take 3
```

Comment:
Hmmm, this doesn't work so well on mine — does it for you?

I think maybe we write the `Ctrl-d` to stdout?

```
prqlc compile | duckdb
from `penguins.csv`
take 3
Error: near line 1: Parser Error: syntax error at or near "Enter"
LINE 1: Enter PRQL, then ctrl-d:
        ^
```

(Is there a way of retaining the query? I worry this approach is OK but loses the query on each run. But it's still reasonable to show the example)

Reply:
It seems that the `prqlc` you are using was not built from this PR. (the prompt message is different)

Reply:
> Is there a way of retaining the query?

I believe the query will remain visible in the terminal as shown in this README, and I have verified that it works with bash and fish on Linux and PowerShell on Windows.

Reply:
> It seems that the `prqlc` you are using was not built from this PR. (the prompt message is different)

Sorry, you're correct, apologies

Reply:
> I believe the query will remain visible in the terminal as shown in this README, and I have verified that it works with bash and fish on Linux and PowerShell on Windows.

Yes!

A total aside — something @snth and I discussed on the dev call was building a TUI, similar to the playground, where typing a query would pipe through to duckdb on each keystroke, could be a good way of exploring a dataset from the terminal